dev/how-to/maintain/tidb-binlog.md

```diff
@@ -1,11 +1,13 @@
 ---
-title: TiDB-Binlog Cluster Operations
-summary: Learn how to operate the cluster version of TiDB-Binlog.
+title: TiDB Binlog Cluster Operations
+summary: Learn how to operate the cluster version of TiDB Binlog.
 category: reference
-aliases: ['/docs/tools/binlog/operation/']
+
+aliases: ['/docs/tools/binlog/operation/','/docs-cn/dev/reference/tools/tidb-binlog/operation/']
```

Comment:
Did you mean to have `/docs-cn/` in the reference here?

Reply:
Nope, it's a mistake and should be /docs-cn/dev/reference/tools/tidb-binlog/operation/ here.  Iw will correct it.

Reply:
Fixed in https://github.com/pingcap/docs/pull/1251/commits/fce70f44743e56daebe705457b569c2fcedbb6a1